### DISCUSSION TITLE
微信公众号批量下载工具2.6 支持Windows、Macos

### DISCUSSION THREAD

**Discussion by qiye45** [qiye45/wechatDownload#112] [Q&A] [answered]
2.6更新 修复下载空白的问题 https://github.com/qiye45/wechatDownload/issues/111 ，支持图文格式下载，支持以公众号文章标题命名视频、音频和封面啦

<hr /><em>This discussion was created from the release <a href='https://github.com/qiye45/wechatDownload/releases/tag/'>微信公众号批量下载工具2.6 支持Windows、Macos</a>.</em>

**Comment by yuleshow** [comment]
2.6 mac版下載後無法運行，說文件已損壞

**Reply by qiye45** [reply]
试试打开app里面的wechatdownload

![image_2024-06-01_20-08-30](https://github.com/qiye45/wechatDownload/assets/138199658/b7815fdd-5f8b-4532-a631-0263e8c44cf1)

**Comment by kangta1027** [comment]
2.5开始剪切板监控功能无效，2.6亦如此

**Reply by qiye45** [reply]
添加截图以帮助解释你的问题。

**Reply by kangta1027** [reply]
![PixPin_2024-06-04_11-55-48](https://github.com/qiye45/wechatDownload/assets/141288702/89f6d3fd-76c9-4770-884a-7d5f6d0b8c83)
![PixPin_2024-06-04_11-54-29](https://github.com/qiye45/wechatDownload/assets/141288702/8c4a85e4-8364-4054-9e23-486c7a632543)
分别对比2.4》2.6版本，自动监听剪切板2.4版本能识别，2.5-2.6都无法识别

**Reply by qiye45** [reply]
下个版本我修一下(=・ω・=)

**Comment by TingyuAI** [comment]
<img width="245" alt="image" src="https://github.com/qiye45/wechatDownload/assets/155154385/25bdb0e4-6a7f-48ad-8279-803f52db039d">
<img width="553" alt="image" src="https://github.com/qiye45/wechatDownload/assets/155154385/ad86cc81-6f2d-4f68-83d2-de37c4eb2e99">
2.6 mac版下载后无法使用
<img width="252" alt="image" src="https://github.com/qiye45/wechatDownload/assets/155154385/58e63cda-cffb-489f-907d-5ea554880583">

**Reply by qiye45** [reply]
要苹果m芯片的

**Reply by TingyuAI** [reply]
好吧。后续有可能支持英特尔嘛。。

**Reply by qiye45** [reply]
兼容可能比较难，还得找个Intel的电脑，可以看看这个哟 https://github.com/qiye45/wechatDownload/issues/107

**Comment by laowei321** [comment]
工具抓几个了就闪退，是被查杀了么？

**Reply by qiye45** [reply]
截个图看看，或者录个视频吧

**Reply by laowei321** [reply]
https://github.com/qiye45/wechatDownload/assets/38400110/34f71aed-9077-439e-b35b-47352d07c558

这是录屏

**Reply by qiye45** [reply]
奇怪了，试试旧版本会闪退吗

**Reply by laowei321** [reply]
旧版本在哪下载？

**Reply by qiye45** [reply]
右边的release

**Comment by sjq597** [comment]
M2 Pro芯片不支持吗，运行不了
![image](https://github.com/qiye45/wechatDownload/assets/4928331/c05f07df-53e7-4b6c-96ee-16027f91242c)
<img width="260" alt="image" src="https://github.com/qiye45/wechatDownload/assets/4928331/a7848058-8097-49e9-861c-d836d9493d4e">
话说这个是什么导致的啊

**Reply by qiye45** [reply]
试试打开app里面的wechatdownload

![image_2024-06-01_20-08-30](https://github.com/qiye45/wechatDownload/assets/138199658/b7815fdd-5f8b-4532-a631-0263e8c44cf1)

**Comment by Ann-9527** [comment]
<img width="460" alt="截屏2024-06-15 09 35 33" src="https://github.com/qiye45/wechatDownload/assets/153164356/7e4a3999-841d-4962-98b7-df1d63575aaf">
我的也是...

**Reply by qiye45** [reply, marked as answer]
macos遇到文件已损坏的问题，可以试试这个哟 https://github.com/qiye45/wechatDownload/issues/107

**Comment by skyding717** [comment]
我放了一个公众号文章链接，但那个识别公众号id是个灰色图标，不知为啥。2.6版本
![22](https://github.com/qiye45/wechatDownload/assets/140905685/fb9e6eb4-d41c-4b58-b0b1-d5adf3838cfd)

**Comment by skyding717** [comment]
我放了一个公众号文章链接，但那个识别公众号id是个灰色图标，不知为啥。2.6版本
![22](https://github.com/qiye45/wechatDownload/assets/140905685/fb9e6eb4-d41c-4b58-b0b1-d5adf3838cfd)

**Reply by WongCM** [reply]
我的也是这个问题，获取公众号id的按钮无法点击

**Reply by CanuteT** [reply]
同灰色，有进一步消息吗

**Comment by liwenjie160** [comment]
下载设置这里，只勾选"**导出PDF**"的情况下，下载完成后只有文件封面，没有PDF文件。
如果把“**导出HTML文件**”也一并勾选上，文件下载就是正常的，下载完毕后HTML文件和PDF都存在。
 这个不是什么大问题，也不怎么影响使用，只不过最新的3.1的版本也存在，前来反馈一下。

![image](https://github.com/user-attachments/assets/65cc0a1b-37df-4b51-9d85-bb04f61d7ac6)
![image](https://github.com/user-attachments/assets/f4447fd2-aa49-449d-bd31-95340c11831d)

**Reply by qiye45** [reply]
选上下载HTML文件就好啦－O－

**Comment by yyu007** [comment]
能放个安装文件的百度网盘吗，夸克很难用，还要下软件

**Comment by pengyw2020** [comment]
夸克网盘失效了，恳请楼主补下链接，谢谢。

**Comment by tianzhi0hao** [comment]
你好，感谢分享工具。MacOS 15.2，M1Pro， 使用2.6版本，粘贴微信文章，“1.获取公众号id”按钮是灰色的，无法批量下载。帮忙看一下呢，谢谢。
<img width="983" alt="截屏2024-12-16 22 20 00" src="https://github.com/user-attachments/assets/a9a38b5c-77bb-485f-a404-67422db62166" />

**Reply by qiye45** [reply]
mac还没更新这个

**Reply by tianzhi0hao** [reply]
可以问一下有更新计划么，找了几个工具，能用的就这个最好用了🥹windows上用过一次就爱上了

**Reply by qiye45** [reply]
我看看吧，先用window咯
